.vale/fixtures/RedHat/CaseSensitiveTerms/testinvalid.adoc

```diff
@@ -145,6 +145,7 @@ Red Boot
 Red Hat Proxy
 Red Hat Satellite
 Redboot
+RESTEASY
```

Comment:
Does this file need to include all the invalid forms of RESTEasy you're adding to .vale/styles/RedHat/CaseSensitiveTerms.yml?

Reply:
Good catch. Fixed!